src/codeql.ts

```diff
@@ -449,17 +449,28 @@ export function getSarifResultCount(sarif: Sarif): number {
   return count;
 }
 
+/**
+ * Names of result sets that can be considered the "default" result set
+ * and should be used when calculating number of results and when showing
+ * results to users.
+ */
+const KNOWN_RESULT_SET_NAMES: string[] = ["#select", "problems"];
```

Comment:
This list is also setting the order by which we'll look for result sets. It would be great if the name could make that obvious (although I'm struggling to think of a good suggestion)